README.md

```diff
@@ -171,29 +171,57 @@ wasm-pack test --headless --chrome ./<crate-folder-to-test>
 cargo nextest run --features test-all-cipher
 ```
 
-### Platform-specific testing
+### Platform-specific tests for Kotlin/JVM
 
-### Kotlin/Android
+Build the JVM target every timee the Rust code changes:
 
-* Take the steps to compile for Kotlin/Android
-* Then:
-```ignore
-cd crypto-ffi/bindings
-./gradlew test
+```sh
+# substitute with `jvm-darwin` on OSX
+core-crypto/crypto-ffi$ cargo make jvm-linux
+```
+
+Then run the tests each time the wrapper or wrapper tests change
+
+```sh
+core-crypto/crypto-ffi/bindings$ ./gradlew jvm:build -x lint -x lintRelease
+```
+
+### Platform-specific tests for Android
+
+Build the Android target every timee the Rust code changes:
+
+```sh
+core-crypto/crypto-ffi$ cargo make android
+```
+
+Then run the tests each time the wrapper or wrapper tests change
+
+```sh
+core-crypto/crypto-ffi/bindings$ ./gradlew android:build -x lint -x lintRelease
 ```
 
 ### Swift/iOS
 
 *No E2E testing is available as of now on Swift.*
 
-### WASM/Web
+### Platform-spcecific tests for WASM/Web
 
-* Take the steps to compile for WASM/Web (only necessary if the rust wasm code changed, building the wrapper is 
-* included in below step).
-* Then:
-```ignore
-cd crypto-ffi
-bun run wdio
+Build the WASM target every timee the Rust code changes:
+
+```sh
+core-crypto/crypto-ffi$ cargo make wasm
+```
+
+Then run the tests each time the wrapper or wrapper tests change
+
+```sh
+core-crypto/crypto-ffi$ bun run wdio
```

Comment:
It might be useful to mention the need for `bun run install` somewhere, since that's required before running tests.